### ISSUE TITLE
Migrate Meetup.com syncing to OAuth 2.0

### ISSUE THREAD

**Issue by nelsonjchen** [desertpy/desertpy-pelican#28]
• API Keys will be replaced by OAuth: We will be removing API keys on August 15, 2019 and requiring you to authenticate with OAuth.
• Move to OAuth soon for continued free API access: Until August 15, members will be able to apply for OAuth access free of charge. After August 15, anyone who wants to apply for API access through OAuth will need to have a Meetup Pro account in order to do so.

**Comment by godber** [comment]
Good catch Nelson.  Do you have time to handle this over the next few weeks and do you need something from me?

**Comment by nelsonjchen** [comment]
I have time to handle this. The API is being locked down though with approvals and crap. There's also technical blockers like no libraries supporting this API.

Sure feels a bit like the Extinguish part of Embrace, Extend, and Extinguish and the Twitter API lockdown. Hmm.

**Comment by nelsonjchen** [comment]
Oh, they approved the "application". Ok, it's a little progress. Not sure how to use the new API though, the libraries don't seem to support it yet.

**Comment by nelsonjchen** [comment]
Hmm, the situation to try and use in in a headless server is pretty dire. Might need to do some screen scraping puppeteer action. At least until the Android App's API is RE'd.

**Comment by nelsonjchen** [comment]
I think I'll put out a pull request with the posts from the current API and commit them for now.

**Comment by nelsonjchen** [comment]
Done on #29. I'll work on the unofficial public API next.
